### ISSUE TITLE
Build failed for JHipster UAA ver5.0.0. Quotes missing.

### ISSUE THREAD

**Issue by binakot** [jhipster/generator-jhipster#7836]
Just updated to JHipster 5.0.0.
Generate new JHipster UAA and type `./gradlew`.
Output:

```bash
$ ./gradlew
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/binakot/Projects/Tiketer/uaa-server/build.gradle' line: 190

* What went wrong:
Could not compile build file '/Users/binakot/Projects/Tiketer/uaa-server/build.gradle'.
> startup failed:
  build file '/Users/binakot/Projects/Tiketer/uaa-server/build.gradle': 190: expecting anything but ''\n''; got it anyway @ line 190, column 52.
     ngframework.retry:spring-retry
                                   ^

  1 error
```

Here 189, 190, 191 lines:

```groovy
compile "org.springframework.cloud:spring-cloud-starter-netflix-hystrix"
compile "org.springframework.retry:spring-retry
compile "io.micrometer:micrometer-core"
```

You miss `"` at end of line.

**Comment by xetys** [comment]
I will check that asap.

Ivan Muratov <notifications@github.com> schrieb am Fr., 22. Juni 2018,
20:36:

**Comment by binakot** [comment]
When I add missed quotes, the project successfully compiled and run.

**Comment by pascalgrimaud** [comment]
@binakot : do you want to contribute and PR ?

**Comment by binakot** [comment]
@pascalgrimaud sure, why not 😊

**Comment by deepu105** [comment]
This is already fixed in master #7820

**Comment by binakot** [comment]
Sorry, missed this one 👍
